src/frontend/qt_sdl/PlatformConfig.cpp

```diff
@@ -112,6 +113,11 @@ ConfigEntry PlatformConfigFile[] =
     {"HKKey_FastForwardToggle",   0, &HKKeyMapping[HK_FastForwardToggle],   -1, NULL, 0},
     {"HKKey_SolarSensorDecrease", 0, &HKKeyMapping[HK_SolarSensorDecrease], -1, NULL, 0},
     {"HKKey_SolarSensorIncrease", 0, &HKKeyMapping[HK_SolarSensorIncrease], -1, NULL, 0},
+    {"HKKey_GuitarGripGreen",     0, &HKKeyMapping[HK_GuitarGripGreen],     -1, NULL, 0},
+    {"HKKey_GuitarGripRed",       0, &HKKeyMapping[HK_GuitarGripRed],       -1, NULL, 0},
+    {"HKKey_GuitarGripYellow",    0, &HKKeyMapping[HK_GuitarGripYellow],    -1, NULL, 0},
+    {"HKKey_GuitarGripBlue",      0, &HKKeyMapping[HK_GuitarGripBlue],      -1, NULL, 0},
+
```

Comment:
Stray newline here.

Reply:
Currently working on resolving that issue right now. Should be ready later today!

Reply:
I've already resolved this issue, rzumer!

Reply:
I still see one unnecessary line break added between this block and the next.